runtime/litmus/src/lib.rs

```diff
@@ -783,6 +784,12 @@ impl pallet_sidechain::Config for Runtime {
 	type EarlyBlockProposalLenience = ConstU64<100>;
 }
 
+impl pallet_teeracle::Config for Runtime {
```

Comment:
Strange, this should be in `dev` already.

Reply:
Yes. It should be. Git problem.

Reply:
One sec, I just noticed it was from `tee-dev` to `dev`. Maybe we shouldn't do it right now, as these two branches diverge. If you need something badly in the `tee-dev` branch (due to incompatible tee-worker development), it's better to submit a PR to `tee-dev` branch directly.

We will merge `dev` to `tee-dev` once the upstream is updated to v0.9.29